<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>531</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>171</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 27, 39);
alternate-background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="MainlogoWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>169</width>
       <height>389</height>
      </rect>
     </property>
     <widget class="QWidget" name="logoWidgetCAN" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>171</width>
        <height>60</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="companyName_2">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:white;</string>
         </property>
         <property name="text">
          <string>CAN</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButtonHome">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>60</y>
        <width>181</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #0c9089; color: black; font-weight: bold; padding: 8px 16px; border-radius: 0px;
</string>
      </property>
      <property name="text">
       <string>HOME</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonHelps">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #8d8d8d; color: black; font-weight: bold; padding: 8px 16px; border-radius: 0px;
border-color: rgb(255, 255, 255);border: 1px solid white;
</string>
      </property>
      <property name="text">
       <string>Helps</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCar">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #8d8d8d; color: black; font-weight: bold; padding: 8px 16px; border-radius: 0px;
border-color: rgb(255, 255, 255); border: 1px solid white;
 
</string>
      </property>
      <property name="text">
       <string>Cars</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonMazda">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: lightgray;
    border-radius: 0px;
 	color: black;
}

QPushButton:hover {
    background-color: gray;
     color: black;
}

</string>
      </property>
      <property name="text">
       <string>Mazda</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonHonda">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: lightgray;
    border-radius: 0px;
 	color: black;
}

QPushButton:hover {
    background-color: gray;
    color: black;
}
</string>
      </property>
      <property name="text">
       <string>Honda</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonHyundai">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: lightgray;
    border-radius: 0px;
 	 color: black;
}

QPushButton:hover {
    background-color: gray;
 	color: black;
}
</string>
      </property>
      <property name="text">
       <string>Hyundai</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonTesla">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: lightgray;
    border-radius: 0px;
 	color: black;
}

QPushButton:hover {
    background-color: gray;
    color: black;
}
</string>
      </property>
      <property name="text">
       <string>Tesla</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonDocuments">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>230</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: lightgray;
    border-radius: 0px;
 	color: black;
}

QPushButton:hover {
    background-color: gray;
    	color: black;
}
</string>
      </property>
      <property name="text">
       <string>Documents</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonVideo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>250</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: lightgray;
    border-radius: 0px;
 	color: black;
}

QPushButton:hover {
    background-color: gray;
 	color: black;
}
</string>
      </property>
      <property name="text">
       <string>Video</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonDevices">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>121</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: lightgray;
    border-radius: 0px;
 	color: black;
}

QPushButton:hover {
    background-color: gray;
 	color: black;
}
</string>
      </property>
      <property name="text">
       <string>Devices</string>
      </property>
     </widget>
     <zorder>pushButtonHyundai</zorder>
     <zorder>logoWidgetCAN</zorder>
     <zorder>pushButtonHome</zorder>
     <zorder>pushButtonHelps</zorder>
     <zorder>pushButtonCar</zorder>
     <zorder>pushButtonMazda</zorder>
     <zorder>pushButtonHonda</zorder>
     <zorder>pushButtonTesla</zorder>
     <zorder>pushButtonDocuments</zorder>
     <zorder>pushButtonVideo</zorder>
     <zorder>pushButtonDevices</zorder>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>481</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">image: url(:/Assets/Resized_Futuristic_Car_Illustration.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>340</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Car_logos/TESLA LOGO.png);
border: 2px solid blue;
 border-radius: 5px; </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>340</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Car_logos/HONDA LOGO.png);
border: 2px solid blue;
 border-radius: 5px; </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>250</y>
      <width>151</width>
      <height>78</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Assets/result_mazda-logo-2018-vertical-download.png);
border: 2px solid blue;
 border-radius: 5px; </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>250</y>
      <width>151</width>
      <height>78</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/Car_logos/hyundai logo.png);
border: 2px solid blue;
 border-radius: 5px; </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>744</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
